templates/publishing/src/blog-app.html

```diff
@@ -40,6 +40,7 @@
 
       :host {
         --app-primary-color: #404040;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
I propose:

```
/* NOTE: Both values are needed, since some phones require the value to be `transparent`. */
-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-webkit-tap-highlight-color: transparent;
```

Reference: https://github.com/PolymerElements/paper-fab/pull/48#discussion_r50937454